app/tests/test_reporter.py

```diff
@@ -0,0 +1,84 @@
+import os
+import pytest
+from pyaml_env import parse_config
+
+from app.reporter import update_cost_reporter, get_current_and_previous_months_dates
+
+
+@pytest.fixture
+def aws_static_costs():
+    return {"current_month": 700, "previous_month": 800}
+
+
+@pytest.fixture
+def aws_accounts_file():
+    aws_accounts_filename = "app/tests/manifests/test_accounts.yaml"
+    os.environ["AWS_COST_REPORTER_CONFIG"] = aws_accounts_filename
+
+    return aws_accounts_filename
+
+
+@pytest.fixture
+def aws_account_names(aws_accounts_file):
+    return [
+        account_name
+        for account_name, account_data in parse_config(path=aws_accounts_file)[
+            "accounts"
+        ].items()
+    ]
+
+
+@pytest.fixture
+def expected_cost_report_message(aws_static_costs, aws_account_names):
+    this_month_start, this_month_end, last_month_start, last_month_end = (
+        get_current_and_previous_months_dates()
+    )
+    return f"""{aws_account_names[0]}:
```

Comment:
```
msg = ""
    for aws_account_name in aws_account_names:
        msg += f"{aws_account_name}:\n"
        msg += f"\t[{this_month_start}/{this_month_end}]{float(aws_static_costs['current_month']): .2f}$\n"
        msg += f"\t[{last_month_start}/{last_month_end}]{float(aws_static_costs['previous_month']): .2f}$\n"

    return msg
```

Reply:
https://github.com/RedHatQE/aws-cost-reporter/pull/32#discussion_r1586045376
raising this comment to be in sync

Reply:
done